entity-framework/toc.yml

```diff
@@ -66,6 +66,8 @@
         href: core/get-started/install/index.md
       - name: ASP.NET Core tutorial >>
         href: /aspnet/core/data/ef-rp/intro
+      - name: Get started with Xamarin and EF Core
```

Comment:
It's already in the "Get Started" section so this can be simplified with "Xamarin tutorial"